sklearn/metrics/_plot/roc_curve.py

```diff
@@ -123,6 +129,24 @@ def plot(self, ax=None, *, name=None, **kwargs):
         if ax is None:
             fig, ax = plt.subplots()
 
+        # Set limits of axes to [0, 1] and fix aspect ratio to squared
+        ax.set_xlim((0, 1))
+        ax.set_ylim((0, 1))
+        ax.set_aspect(1)
```

Comment:
Let's move this change in another PR. We will need an additional entry in the changelog since we are fixing/improving the rendering.

Reply:
It should be shared with PR and ROC curve